ReactWindows/ReactNative/Views/WebView/Events/WebViewLoadingErrorEvent.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.UIManager.Events;
+using System;
+using Windows.Web;
+
+namespace ReactNative.Views.WebView.Events
+{
+    class WebViewLoadingErrorEvent : Event
+    {
+        private readonly double _code;
+        private readonly string _description;
+
+        public WebViewLoadingErrorEvent(int viewTag, WebErrorStatus error)
+            : base(viewTag, TimeSpan.FromTicks(Environment.TickCount))
+        {
+            _code = (double)error;
+            _description = ErrorString(error);
+        }
+
+        public override string EventName
+        {
+            get
+            {
+                return "topLoadingError";
+            }
+        }
+
+        public override void Dispatch(RCTEventEmitter eventEmitter)
+        {
+            var eventData = new JObject
+                {
+                    { "target", ViewTag },
+                    { "code", _code },
+                    { "description", _description },
+                };
+
+            eventEmitter.receiveEvent(ViewTag, EventName, eventData);
+        }
+
+        private string ErrorString(WebErrorStatus status)
+        {
+            switch (status)
+            {
+                case WebErrorStatus.Unknown: return "An unknown error has occurred.";
```

Comment:
Also, is there not already some default stringify method for WebErrorStatus?

---
In reply to: [60680625](https://github.com/ReactWindows/react-native/pull/372#discussion_r60680625) [](ancestors = 60680625)